wgsl/index.bs

```diff
@@ -1039,15 +1039,44 @@ switch_stmt
   : SWITCH paren_rhs_stmt BRACKET_LEFT switch_body+ BRACKET_RIGHT
 
 switch_body
-  : CASE const_literal COLON BRACKET_LEFT case_body BRACKET_RIGHT
+  : CASE case_selectors COLON BRACKET_LEFT case_body BRACKET_RIGHT
   | DEFAULT COLON BRACKET_LEFT case_body BRACKET_RIGHT
 
+case_selectors
+  : const_literal (COMMA const_literal)*
+
 case_body
   :
   | statement case_body
   | FALLTHROUGH SEMICOLON
 </pre>
 
+A switch statement transfers control to one of a set of case clauses, or to a default clause,
+depending the evaluation of a selector expression of a scalar integer type.
+
+If the selector value equals a value in a case selector list, then control is transferred to
+the body of that case clause.
+If the selector value does not equal any of the case selector values, then control is
+transferred to the `default` clause.
+
+Each switch statement must have exactly one default clause.
+
+A literal value must not appear more than once in the case selectors for a switch statement.
+
+The case selector values must have the same type as the selector expression.
+
+Note: A literal value must not be duplicated within a case clause, and must not appear in more
+than one case selector list.
+
+Note: The value of the literal is what matters, not the spelling. For example `0` and `00` both
+denote the zero value.
+
```

Comment:
If you're going to give examples, maybe also provide a decimal v hex one where it is a more different spelling?

Reply:
Added 0x0000 as another example spelling of 0.